rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, person, book
```

Comment:
- [ ] You may prefix person and book with colon like this `:person, :book` in order to write the correct syntax.

Reply:
oh, thanks. that was actually an error. fixing right away